packages/htmlviewer/src/widget.tsx

```diff
@@ -33,6 +33,22 @@ const RENDER_TIMEOUT = 1000;
  */
 const CSS_CLASS = 'jp-HTMLViewer';
 
+const UNTRUSTED_LINK_STYLE = (options: { warning: string }) => `<style>
+a {
+  cursor: not-allowed !important;
+}
+a:hover::after {
```

Comment:
Normal links work and do not need this. Only popup links are restricted.

```suggestion
a[target="_blank"] {
  cursor: not-allowed !important;
}
a[target="_blank"]:hover::after {
```

Reply:
Thanks @krassowski
I was looking at the [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes) and apparently 4 tags support `target`: `a`, `area`, `base` and `form`. I guess we should cover `a[target="_blank"]`, `area[target="_blank"]` and `form[target="_blank"]` (as the one for `base` is anyway overwritten) as the documentation for `allow-popups` does not specify any tag - only `target="blank_"`?

Reply:
Right, thank you! I also see `formtarget`:

> This value can be overridden by a [formtarget](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#formtarget) attribute on a [`<button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button), [`<input type="submit">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit), or [`<input type="image">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/image) element.